### PR TITLE
map: interpret POI taps on Android

### PR DESCRIPTION
**Pull request by stevekuznetsov**
x-ref for issue on exact coordinates: https://github.com/react-native-maps/react-native-maps/issues/2496

### CODE REVIEW

**Comment by stevekuznetsov** [comment]
Fixes https://github.com/NWACus/avy/issues/77
Fixes

**Comment by stevekuznetsov** [comment]
Fixes https://github.com/NWACus/avy/issues/604

**Comment by stevekuznetsov** [comment]
| before | after |
| - | - |
| https://github.com/NWACus/avy/assets/7328370/3c077712-aa13-47ae-ac8b-191f0607a601 | https://github.com/NWACus/avy/assets/7328370/d6d21a3f-74f8-4820-b995-f90a433a49e8 |

File: components/AvalancheForecastZoneMap.tsx [hunk 1]
```diff
@@ -62,6 +63,29 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
...
+          navigation.navigate('forecast', {
+            zoneName: matchingZones[0].properties.name,
+            center_id: center,
+            forecast_zone_id: matchingZones[0].id,
+            requestedTime: formatRequestedTime(requestedTime),
+          });
+        } else {
+          setSelectedZoneId(matchingZones[0].id);
+        }
+      }
```
Comment (floatplane) [inline components/AvalancheForecastZoneMap.tsx:85]: @stevekuznetsov I would have changed these lines to 
```
onPressPolygon(matchingZone[0])
```
to minimize copy-pasta

**Pull request merged**